operator/controllers/mlops/suite_test.go

```diff
@@ -46,7 +45,7 @@ func TestAPIs(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(
 		t,
 		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}},
```

Comment:
this library doesnt exist anymore in `controller-runtime` and I dont think it is required for testing.

Reply:
appreciate the comment, helpful to know why the change was needed!